### PR TITLE
doc: trivial update for `desugar` -> substitution

### PR DESCRIPTION
**Pull request by mvo5**
Trivial followup for PR#120

### CODE REVIEW

**Review by achilleas-k — APPROVED** [review]

File: doc/03-omnifest/01-directive.md [hunk 1]
```diff
@@ -73,7 +73,7 @@ otk.include: ${variable}
 ```
 
 If a `${}` appears in a `str` value then its string value as it appears
-in `otk.define` is replaced into the string. Note that using the sugared form
+in `otk.define` is replaced into the string. Note that substitutions
 in this form requires the value to be a string in the `otk.define`.
```
Comment (achilleas-k) [inline doc/03-omnifest/01-directive.md:77]:
```suggestion
in `otk.define` is replaced into the string. Note that substitutions
in this form require the value to be a string in the `otk.define`.
```

**Review by achilleas-k — CHANGES_REQUESTED** [review]
Approved too fast.

File: doc/03-omnifest/01-directive.md (hunk 1 above)
Comment (mvo5) [reply doc/03-omnifest/01-directive.md:77]: Hm, hm, I'm not a native speak but to me both sound okay? Fine committing this though (as both sound okay to me :)

**Review by achilleas-k — APPROVED** [review]
Typo in the second commit message, but I'm good to merge if you are.

**Pull request merged**